ctfpad/models.py

```diff
@@ -637,7 +637,7 @@ def files(self):
     def jitsi_url(self):
         return f"{JITSI_URL}/{self.ctf.id}--{self.id}"
 
-    def save(self):
+    def save(self, **kwargs):
```

Comment:
`kwargs` is unused, why the change?

Reply:
Actually is used. Because if it is missing we get:

<img width="567" alt="image" src="https://github.com/hugsy/ctfpad/assets/9267793/2e3e9101-efa8-4f08-bc90-9c7098d3d4bc">

Please don't ask me why, I failed to figure out exactly why this is passed but it is...

Reply:
Weird, ok